CONTRIBUTING.md

```diff
@@ -20,10 +20,11 @@ Guidelines for pull requests:
 
 ### Pull Requests Including Icons
 
-#### Guidelines
+#### Guidelines & Templates
 
 Please make sure you follow the icon guidelines, that should be followed to keep quality and consistency when making icons for Lucide
-Read it here: [ICON_GUIDELINES](docs/ICON_DESIGN_GUIDE.md)
+
+Read it here: [ICON_GUIDELINES](docs/ICON_DESIGN_GUIDE.md). You can find a template and instructions on how to implement those guidelines with Adobe Illustrator and Inkscape in the */docs* folder of the project.
```

Comment:
Would've made a seperate section for instructions relating to particular software.

Reply:
Agree. 
Adjusted in the latest commit 👍🏽